website/lib/consent-manager-config.js

```diff
@@ -0,0 +1,76 @@
+const isProd = process.env.NODE_ENV === 'production'
+
+const segmentWriteKey = isProd
+  ? 'AjXdfmTTk1I9q9dfyePuDFHBrz1tCO3l'
+  : '0EXTgkNx0Ydje2PGXVbRhpKKoe5wtzcE'
```

Comment:
Would it be possible and a good idea to put the prod one in netlify ? Can those keys be used only to write data ?